src/app/View.elm

```diff
@@ -33,6 +34,17 @@ firstPlayer players =
             "No Players Found"
 
 
-tableView : Html Msg
-tableView =
-    Html.map TableMsg (Table.view Table.init)
+tableView : List Player -> Dice.Cup -> Html Msg
+tableView players cup =
+    let
+        -- we have to fake ids until we have them
+        idAndPlayerTo3Tuple id { playerName, diceCount } =
+            ( id, playerName, diceCount )
+
+        ids =
+            List.range 1 <| List.length players
+
+        playersTuples =
+            List.map2 idAndPlayerTo3Tuple ids players
+    in
+        Table.view Types.TableAction <| Table.initWithPlayersAndCup playersTuples cup
```

Comment:
A view function should simply be taking state and displaying it. Why are we initialising a table in the view function?

My first impression is `tableView : Game.Table -> Html Msg`